model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -404,17 +407,72 @@ def _get_next_perf_analyzer_config(self, model_variant_name: str,
         perf_analyzer_config.update_config_from_profile_config(
             model_variant_name, self._config)
 
-        model_batch_size = dimension_values.get("max_batch_size", 1)
-        instance_count = dimension_values.get("instance_count", 1)
+        dimension_batch_size = int(dimension_values.get("max_batch_size", 1))
```

Comment:
can you hide all of the retrieval + calculation in the function? So we don't have half of it done here with a temporary value?